### DISCUSSION TITLE
Problem: obj loss start increasing

### DISCUSSION THREAD

**Discussion by CesareDavidePace** [ultralytics/yolov5#10474] [Q&A]
Hi,
I have a problem regarding obj loss.
In the guide it says: Hyperparameters. Default hyperparameters are in hyp.scratch-low.yaml. We recommend you train with default hyperparameters first before thinking of modifying any. In general, increasing augmentation hyperparameters will reduce and delay overfitting, allowing for longer trainings and higher final mAP. Reduction in loss component gain hyperparameters like hyp['obj'] will help reduce overfitting in those specific loss components. For an automated method of optimizing these hyperparameters, see our Hyperparameter Evolution Tutorial.

This is because at a certain time the obj loss starts to grow, so I tried both to put a lower gain and to increase the augmentation but the final result is worse.
![results](https://user-images.githubusercontent.com/104144011/207008222-3f2e2a07-4ddf-4416-9e2d-58e7ca2c46c4.png)

**Comment by glenn-jocher** [comment]
@CesareDavidePace yes, obj is overfitting around epoch 30, which is a common problem. You'll just have to continue experimenting, or even start training from scratch, i.e. `python train.py --cfg yolov5s.yaml --weights ''`

**Reply by CesareDavidePace** [reply]
can changing obj_pw change things?

**Reply by glenn-jocher** [reply]
@CesareDavidePace yes reducing that might help, i.e. to 0.5

**Reply by CesareDavidePace** [reply]
ok stay test and tomorrow I'll update you. I thank you

**Comment by CesareDavidePace** [comment]
Hello,
I tried as you said, I did two tests:
1) obj = 0.3 obj_pw = 0.3
![results](https://user-images.githubusercontent.com/104144011/207262821-590bdf72-0935-4a3d-9a49-4919e2691268.png)

result on test:
![confusion_matrix](https://user-images.githubusercontent.com/104144011/207263099-cbec8659-a387-42fd-afb5-de82cba9fc4e.png)

2) obj = 0.3 obj_pw 2.0
![results2](https://user-images.githubusercontent.com/104144011/207263447-362fefe3-f817-40af-b808-9cf24e519700.png)

result on test:

![confusion_matrix2](https://user-images.githubusercontent.com/104144011/207265285-a02fca08-f613-4932-9aa2-9095257a59d6.png)

How is it possible that the test is better in the second even if it starts to overfit? I checked and there are no objects in common.

Even F1-score i better
